src/actions/AppStoreActions.ts

```diff
@@ -38,7 +38,7 @@ export const addCourse = (
         section: { ...section, color: '' },
     };
 
-    AppStore.addCourse(newCourse, scheduleIndex);
+    return AppStore.addCourse(newCourse, scheduleIndex);
```

Comment:
```suggestion
    const addedCourse = AppStore.addCourse(newCourse, scheduleIndex);
    return addedCourse;
```
Optional: more clear what we're doing without requiring context of the `addCourse` method